akka-operator/templates/_helpers.tpl

```diff
@@ -62,3 +62,12 @@ Create the name of the service account to use
 {{- end }}
 {{- end }}
 
+
+{{/*
+Create the configjson for a secret with the lightbend commercial credentials
+*/}}
+{{- define "imagePullSubscriptionSecret" }}
+{{- if ne .Values.lightbendSubscription.username "false" }}
+{{- printf "{\"auths\": {\"%s\": {\"auth\": \"%s\"}}}" .Values.lightbendSubscription.registry (printf "%s:%s" .Values.lightbendSubscription.username .Values.lightbendSubscription.password | b64enc) | b64enc }}
+{{- end }}
+{{- end }}
```

Comment:
This is the second half of https://helm.sh/docs/howto/charts_tips_and_tricks/#creating-image-pull-secrets

Reply:
Note how this JSON I'm building needs the `registry` field of `lightbendSubscription`.